entrypoints/backend.sh

```diff
@@ -8,23 +8,10 @@ else
   rm $prometheus_multiproc_dir/* -rf
 fi
 
-echo "Waiting for postgres... "
-while ! nc -z $SQL_HOST $SQL_PORT
-do
-  sleep 0.69
-done
-echo "Done."
-
 echo "Running migrations..."
 /app/src/manage.py migrate
 echo "Done."
 
-if [ "$LOAD_FIXTURES" ]
```

Comment:
I'm not a fan of removing the fixtures, even though it is unused, it does feel useful. Maybe we could generate fixtures based off 2021 or something, django must have some sort of solution to keep them maintained so we don't end up like last time.

Reply:
Fixtures can still be loaded pretty easily, I just feel that adding custom logic to the entrypoint to load them isn't worth it. RE: Generating fixtures, I think a better solution would just be to have a staging machine somewhere that we can export from. That way migrations,  upgrades etc. will be executed automatically and the database state will always be up-to-date.